recipes/esmf/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $(uname) == Linux ]]; then
+yum install -y gcc-gfortran
```

Comment:
Both are temporary solution IMO. I just like to have it in the `build.sh` script to remind me to remove it as soon as we sort this out, and to avoid conflict when the conda-bot updates the feedstocks.